core/src/main/java/org/elasticsearch/search/suggest/completion/CompletionSuggestion.java

```diff
@@ -35,6 +40,8 @@
 import java.util.Map;
 import java.util.Set;
 
+import static org.elasticsearch.search.suggest.Suggest.COMPARATOR;
```

Comment:
the generic here are aweful. I regret it so much that I added them. We should really clean this up it makes stuff so complicated for no good reason